### PR TITLE
Remove unnecessary semicolons in enum declarations

### PR DESCRIPTION
**Pull request by mnhock**

### CODE REVIEW

File: spring-context/src/main/java/org/springframework/context/annotation/EnableLoadTimeWeaving.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ enum AspectJWeaving {
 		 * is present in the classpath. If there is no such resource, then AspectJ
 		 * load-time weaving will be switched off.
 		 */
-		AUTODETECT;
-	}
+		AUTODETECT
+  }
```
Comment (snicoll) [inline spring-context/src/main/java/org/springframework/context/annotation/EnableLoadTimeWeaving.java:169]: This formatting change should be reverted (same on the three other occurrences.

**Review by snicoll — CHANGES_REQUESTED** [review]
Thanks for the PR but please review the commit to make sure you don't introduce unrelated change. From the GitHub UI I gather the formatting has changed.

**Comment by sbrannen** [comment]
Thanks for the PR.

Unfortunately we don't have a policy stating that we remove semicolons in enum declarations.

In light of that, I am closing this PR.

**Comment by sbrannen** [comment]
Hi @mnhock,

Traditionally we had a mix of enum constant declarations with and without the closing semicolon; however, apparently all of the unnecessary semicolons have been removed over the years except the last four that you updated in this PR.

In light of that, I decided to merge your PR (see a8fa98e2a6ddc28febb07303eab644efe3162025 and 5856d2e54efe9209e415b0547174cd754473d6cc) in order to achieve consistency across the code base, and we'll consider making it an official policy in our Code Style.

Cheers,

Sam

**Comment by mnhock** [comment]
Hi @sbrannen!

Thanks for merging the PR and providing clarity on the consistency across the code base. I appreciate your attention to this and the effort to maintain a unified coding style. 


Thanks to the awesome Spring Team!

**Pull request closed without merging**